test/test.js

```diff
@@ -4,7 +4,7 @@ var test = require('tape'),
     path = require('path'),
     glob = require('glob'),
     nock = require('nock'),
-    mapnik = require('@kartotherian/mapnik'),
+    mapnik = require('mapnik'),
```

Comment:
Are you sure we want to land this on `master`?  I think we become incompatible with current forked mapnik if we do this.

Reply:
But isn't the whole idea to ( also ) get rid of the mapnik fork?

Reply:
Just talked about that in the 1on1. :-)

Reply:
Exciting to see that this is possible!